pkgs/servers/sql/postgresql/packages.nix

```diff
@@ -78,4 +78,6 @@ self: super: {
     rum = super.callPackage ./ext/rum.nix { };
 
     wal2json = super.callPackage ./ext/wal2json.nix { };
+
+    apache_datasketches = super.callPackage ./ext/apache_datasketches.nix { };
```

Comment:
Please add this attribute at the right place, even if the file is not sorted alphabetically, it's maybe better to keep it sorted and move this after `age` attribute.